charts/scalardb-analytics-postgresql/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: scalardb-analytics-postgresql
 description: ScalarDB Analytics with PostgreSQL
 type: application
-version: 1.0.0-SNAPSHOT
+version: 2.0.0-SNAPSHOT
 appVersion: 3.10.3
```

Comment:
ScalarDB Analytics with PostgreSQL does not have the `SNAPSHOT` version at this time.
https://github.com/orgs/scalar-labs/packages/container/scalardb-analytics-postgresql/versions?filters%5Bversion_type%5D=tagged

Before, we discussed it and we will provide the `SNAPSHOT` version in the future.
However, for now, I use the latest stable version for `appVersion` in the chart of ScalarDB Analytics with PostgreSQL.